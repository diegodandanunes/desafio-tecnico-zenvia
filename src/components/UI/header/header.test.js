import React from 'react';
import { render } from '@testing-library/react';

import { Header } from './';

it("renders", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
});
