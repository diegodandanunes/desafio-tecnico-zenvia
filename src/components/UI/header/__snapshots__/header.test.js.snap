// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header>
    <div
      class="sc-AxirZ iXyIpx"
    >
      <div
        class="sc-AxhCb bGeBvo"
      >
        <img
          alt="Logo Diego Nunes"
          src="logo.png"
        />
        <ul>
          <li>
            Desafio t√©cnico 
            <span>
              |
            </span>
             
            <strong>
              Zenvia
            </strong>
          </li>
        </ul>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
